### PR TITLE
[Docs] Database Preparation: Add a step for binding IP address for listening on remote database

### PR DESCRIPTION
**Pull request by bagasme**
As per title.
 
 * Add listen address configuration step
  * Fix numbering order

Signed-off-by: Bagas Sanjaya <bagasdotme@gmail.com>

### CODE REVIEW

File: docs/content/doc/installation/database-preparation.en-us.md [hunk 1]
```diff
@@ -21,15 +21,20 @@ Note: All steps below requires that the database engine of your choice is instal
 
 ## MySQL
 
-1.  On database instance, login to database console as root:
+1.  For remote database setup, you will need to make MySQL listen to your IP address. Edit `bind-address` option on `/etc/mysql/my.cnf` on database instance to:
+
+    ```ini
+    bind-address = 203.0.113.3
```
Comment (6543) [inline docs/content/doc/installation/database-preparation.en-us.md:27]: `203.0.113.3` is this one of your servers?

Comment (6543) [reply docs/content/doc/installation/database-preparation.en-us.md:27]: I would use <public-ip> or some other placeholder ...

**Comment by 6543** [comment]
this addition is somehow obvious ... if you have ever setup a DBMS, does this really impruve the docs?

File: docs/content/doc/installation/database-preparation.en-us.md (hunk 1 above)
Comment (bagasme) [reply docs/content/doc/installation/database-preparation.en-us.md:27]: AFAIK @guillep2k suggested that placeholder IP

Comment (6543) [reply docs/content/doc/installation/database-preparation.en-us.md:27]: this is no blocking point at all, just like to make sure it's not copy-paste from somewhere

**Comment by bagasme** [comment]
@6543 assuming that user had just install DBMS server prior to this docs

**Review by 6543 — APPROVED** [review]
~10 lines more wont hurt me

**Comment by bagasme** [comment]
@6543 such extra changes are fixing numbering order. However, Hugo has auto-numbering feature where I can simply number all lists as `1.` and let Hugo expands the numbering.

I'd like submit another PR after this PR to try that above.

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
🚀

**Pull request merged**